OracleDatabase/RAC/OracleRealApplicationClusters/PODMAN_README.md

```diff
@@ -56,7 +56,7 @@ You must provide block devices shared across the hosts.  If you don't have share
 
 Refer Oracle Database 21c Release documentation:
 
-* [Real Application Clusters Installation Guide for Podman Oracle Linux x86-64](https://support.oracle.com/portal/) to make sure your enviornment meeting all the pre-requisites.
+* [Real Application Clusters Installation Guide for Podman Oracle Linux x86-64](https://support.oracle.com/portal/) to make sure your environment meeting all the pre-requisites.
```

Comment:
This needs to be the direct URL to the document or include instructions on how to find it or the reference must be removed.